app/code/Magento/SalesInventory/Model/Order/ReturnValidator.php

```diff
@@ -16,24 +17,19 @@
  */
 class ReturnValidator
 {
-    /**
-     * @var OrderItemRepositoryInterface
-     */
-    private $orderItemRepository;
-
     /**
      * ReturnValidator constructor.
      * @param OrderItemRepositoryInterface $orderItemRepository
      */
-    public function __construct(OrderItemRepositoryInterface $orderItemRepository)
-    {
-        $this->orderItemRepository = $orderItemRepository;
+    public function __construct(
+        private readonly OrderItemRepositoryInterface $orderItemRepository
```

Comment:
A trailing comma makes future additions easier to review.
```suggestion
        private readonly OrderItemRepositoryInterface $orderItemRepository,
```